flink-runtime/src/main/java/org/apache/flink/runtime/state/filesystem/FsStateBackend.java

```diff
@@ -93,7 +94,7 @@
 	private static final long serialVersionUID = -8191916350224044011L;
 
 	/** Maximum size of state that is stored with the metadata, rather than in files (1 MiByte). */
-	public static final int MAX_FILE_STATE_THRESHOLD = 1024 * 1024;
+	private static final int MAX_FILE_STATE_THRESHOLD = 1024 * 1024;
```

Comment:
This should not be changed, because the class is user-facing API and someone might have used it.

Reply:
Ok, it is `PublicEvolving` so you can keep the change.